### PR TITLE
Adds alert component

### LINKED ISSUE #55: Add Alert component
As a user I would like to see better info | error | warning | success messages 

ac- 
- remove alerts
- add alert component with timeout

### PR DESCRIPTION
**Pull request by dviramontes**
- adds alert component closes #55, connects to #55

@codefordenver/rmfu-team 

The component works like this:
```
[alert :type state-atom state-update-fn]
```
Pass in a *type*  `:error, :warning, :info, :success`
some state and a function to call on timeout.

Refer to `rmfu-ui.admin` for usage.

screen_shot_2015-11-25_at_2_50_06_pm
-----
For some reason ring/compojure is not serving the svg files 
with the correct mime-type so ive disabled the css for this,
it will default to images for the icons until we can figure out what is wrong
with the static file resource config.

### CODE REVIEW

File: resources/public/.htaccess [hunk 1]
```diff
@@ -0,0 +1 @@
+AddType image/svg+xml .svg .svgz
```
Comment (wdoug) [inline resources/public/.htaccess]: `.htaccess`? Does leiningen use this?

Comment (dviramontes) [inline resources/public/.htaccess]: no that was my attempt to get `ring/compojure` to serve `svg`...
(!-_-).•

removing

**Comment by wdoug** [comment]
Are those svg's the same images as the pngs?

File: resources/public/css/style.css [hunk 2]
```diff
@@ -139,4 +139,85 @@ p.articlelink a {
     text-align: center;
     cursor: default;
     background-color: tomato;
+}
+
+/* BEGINS 18F Alert component CSS
+   with minor edits. */
+
+.usa-alert {
```
Comment (wdoug) [inline resources/public/css/style.css]: `usa-alert`? :wink:

**Comment by dviramontes** [comment]
yes, for graceful degradation. Its part of this standard: https://playbook.cio.gov/designstandards/

**Comment by wdoug** [comment]
Okay word

File: resources/public/css/style.css (hunk 2 above)
Comment (dviramontes) [inline resources/public/css/style.css]: thats the name they used, just copied it

**Comment by wdoug** [comment]
I'm excited to have a better means of notifying users than alerts :smile: :+1:

**Comment by dviramontes** [comment]
It basically means, that if it can't render svg for whatever reason it uses an image. But the default is the svg file which are more light weight.

**Comment by dviramontes** [comment]
but like I said in the PR message above, 
ring/compojure is not sending the right content-type for svg :(

it has to do with this issue: https://css-tricks.com/snippets/htaccess/serve-svg-correct-content-type/
maybe you have an idea @wdoug @switzersc ?

**Comment by wdoug** [comment]
Just fyi, you shouldn't need "connects to #55" if you already haves "closes #55"

**Comment by dviramontes** [comment]
Really?, it didn't work for me this one time, so i started using both. But I'll try it again next time.

**Pull request merged**